operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java

```diff
@@ -222,7 +222,7 @@ public void testHostnameStrict() {
                         var curlOutput = K8sUtils.inClusterCurl(k8sclient, namespace, "-s", "--insecure", "-H", "Host: foo.bar", url);
                         Log.info("Curl Output: " + curlOutput);
 
-                        assertTrue(curlOutput.contains("var authServerUrl = 'https://example.com:8443';"));
+                        assertTrue(curlOutput.contains("var authServerUrl = 'https://example.com';"));
```

Comment:
Just a note. This will probably break and will need to be adjusted as soon as we switch to the new Admin Console by default. But it's ok of course, for now.